### PR TITLE
[zh-tw] create the translation of `XMLHttpRequest.open()`

### PR DESCRIPTION
**Pull request by BlueBoy247**
### Description

I created a new page to translate [this page](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open) into Traditional Chinese.

### Motivation

I was looking up information about this statement before and found that this documentation hasn't been translated into Traditional Chinese, so I translated it myself, hoping to help more readers.

### Additional details

No.

### Related issues and pull requests

No.

### CODE REVIEW

File: files/zh-tw/web/api/xmlhttprequest/open/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+{{Specifications}}
+
+## 瀏覽器相容性
+
+{{Compat}}
+
+## 參見
+
+- [Using XMLHttpRequest](/zh-TW/docs/Web/API/XMLHttpRequest_API/Using_XMLHttpRequest)
```
Comment (T34-active) [inline files/zh-tw/web/api/xmlhttprequest/open/index.md:52]:
```suggestion
- [使用 XMLHttpRequest](/zh-TW/docs/Web/API/XMLHttpRequest_API/Using_XMLHttpRequest)
```

File: files/zh-tw/web/api/xmlhttprequest/open/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: "XMLHttpRequest: open()"
+slug: Web/API/XMLHttpRequest/open
+---
+
+{{APIRef('XMLHttpRequest')}}
+
+{{domxref("XMLHttpRequest")}} 物件中的 **`open()`** 方法用來初始化新建立的請求，或重新初始化現有請求。
+
+> **備註：** 為已啟動（已經呼叫 `open()` ）的請求呼叫此方法，相當於呼叫 {{domxref("XMLHttpRequest.abort", "abort()")}}。
```
Comment (T34-active) [inline files/zh-tw/web/api/xmlhttprequest/open/index.md:10]:
```suggestion
> **備註：** 為已啟動（已經呼叫 `open()`）的請求呼叫此方法，相當於呼叫 {{domxref("XMLHttpRequest.abort", "abort()")}}。
```

File: files/zh-tw/web/api/xmlhttprequest/open/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## 瀏覽器相容性
+
+{{Compat}}
+
+## 參見
+
+- [使用 XMLHttpRequest](/zh-TW/docs/Web/API/XMLHttpRequest_API/Using_XMLHttpRequest)
+- {{domxref("XMLHttpRequest.setRequestHeader","setRequestHeader()")}}
+- {{domxref("XMLHttpRequest.send", "send()")}}
+- {{domxref("XMLHttpRequest.abort", "abort()")}}
```
Comment (yin1999) [inline files/zh-tw/web/api/xmlhttprequest/open/index.md:55]:
```suggestion
- {{domxref("XMLHttpRequest")}} 相關方法：{{domxref("XMLHttpRequest.setRequestHeader","setRequestHeader()")}}、{{domxref("XMLHttpRequest.send", "send()")}}、{{domxref("XMLHttpRequest.abort", "abort()")}}
```

**Review by yin1999 — COMMENTED** [review]
请注意翻译内容应贴合原文的格式

**Comment by BlueBoy247** [comment]
了解，感謝指正，未來會多加注意🙇‍♂️

**Review by yin1999 — APPROVED** [review]
Thanks @BlueBoy247

**Pull request merged**